vxquery-core/src/main/java/org/apache/vxquery/metadata/VXQueryCollectionOperatorDescriptor.java

```diff
@@ -49,6 +70,9 @@
     private String[] collectionPartitions;
```

Comment:
Can these variables be final? (I know this is not part of your changes, but it relates to your new variables.)